libs/aws/odc/aws/queues.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+
+def get_queue(queue_name):
```

Comment:
code that creates s3 client supports configuring profile, region code, and accepting credentials, would be nice if this also had these features, there is code that creates session that takes care of all of that, one can then create sqs resource from that session.

Reply:
I think we can wait until we need it to deal with this.